Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractTouch.cs

```diff
@@ -109,8 +124,8 @@ private void Start()
                 return;
             }
 
-            Utilities.SetPlayerObject(this.gameObject, VRTK_PlayerObject.ObjectTypes.Controller);
-            CreateTouchCollider(this.gameObject);
+            Utilities.SetPlayerObject(gameObject, VRTK_PlayerObject.ObjectTypes.Controller);
```

Comment:
I've noticed pretty much all of the changes after this point are to do with removing `this` prefix (or other similar prefixes)

Is there a reason for this?

Personally, I like the `this` prefix for readability as it can tell this isn't a local variable but an inherited monobehaviour.

But, if there is a reason for it, then I think perhaps it should be a separate PR that does it across the board to ensure all classes are consistent.